core/tauri-config-schema/src/main.rs

```diff
@@ -2,4 +2,13 @@
 // SPDX-License-Identifier: Apache-2.0
 // SPDX-License-Identifier: MIT
 
+//! [![](https://github.com/tauri-apps/tauri/raw/dev/.github/splash.png)](https://tauri.app)
+//!
```

Comment:
This crate is never meant to be published, it merely exists to kinda auto regenerate schema files

Reply:
Doesn't mean it can't look pretty 😅 Never know when you might wanna make some internal documentation for it and have developers run `cargo doc` locally

Reply:
agreed